client/utils/auth.py

```diff
@@ -148,22 +153,28 @@ def log_message(self, format, *args):
     return access_token
 
 
-def success_page(server, email):
+def success_page(server, email, access_token):
     """Generate HTML for the auth page - fetch courses and plug into templates"""
-    API = server + '/enrollment?email=%s' % email
-    data = urlopen(API).read().decode("utf-8")
+    API = server + '/api/v3/enrollment/{0}/?access_token={1}'.format(
+        email, access_token)
+    try:
+        data = urlopen(API).read().decode("utf-8")
+    except:
+        log.debug("Enrollment for {} failed".format(email), exc_info=True)
```

Comment:
These changes are just a quick patch so that we maintain the same functionality. At some point the success page will just be a redirect to a page on the server.